src/instructlab/configuration.py

```diff
@@ -307,6 +308,18 @@ def get_default_config():
             mt_bench_branch=_mtbenchbranch(taxonomy_path=DEFAULT_TAXONOMY_PATH),
             mmlu_branch=_mmlubranch(sdg_path=DEFAULT_GENERATED_FILES_OUTPUT_DIR),
         ),
+        evaluate_serve=_serve(
+            model_path=DEFAULT_MODEL_PATH,
+            backend="vllm",
+            llama_cpp=_serve_llama_cpp(
+                gpu_layers=-1,
+                max_ctx_size=5120,  # Larger context needed for mt_bench judgement
+                llm_family="",
+            ),
```

Comment:
Why do we need Llama CPP config? I thought we were only using vLLM for evaluation inferencing

Reply:
I believe instructlab can allow for both.  Is there another place we are not supporting llama-cpp in the cli?

Reply:
This might warrant a broader discussion, but evaluation **could** be done via models served by llama-cpp instead of `vLLM`. To support this case, config for llama-cpp is needed.

Reply:
I'm fine for allowing it, I just wasn't sure if it would work with llama-cpp

Reply:
It does function.  It's quite slow (at least with my setup) though.

Reply:
Can we recommend vLLM somehow? Even if it's just a note in the helptext

Reply:
It defaults to vllm in the config

Reply:
Works for me